### DISCUSSION TITLE
pixhawk geiger counter( radyoactivite detector)

### DISCUSSION THREAD

**Discussion by yahyaeness** [community/community#121546] [Programming Help]
### Select Topic Area

Question

### Body

Hello, I'm a university student working on a project. I want to connect a Geiger counter (radiation detector) to a Pixhawk Cube Orange and obtain radiation data along with coordinate information during flight. Subsequently, I aim to use this data to create a map. I'm not sure how to establish the connection between the Geiger counter and Pixhawk. Additionally, I'm unsure whether I can receive data from Pixhawk via Mission Planner. Is it possible to attach an external sensor to Pixhawk and collect data from it? I lack knowledge on this topic and would greatly appreciate any assistance or information you can provide. Thank you in advance for your help.I'm using translation to write, there may be errors. I apologize.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @yahyaeness ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
